doc/conf.py

```diff
@@ -49,6 +50,21 @@
     "sphinxext.opengraph",
 ]
 
+# Support for `plot::` directives in sphinx 3.2 requires matplotlib 3.1.0 or newer
+if parse(mpl.__version__) >= parse("3.1.0"):
```

Comment:
When we finally update our matplotlib min version, we can remove this version parsing.